## 1.A manufacturer of print-heads for personal computers is interested in estimating the mean durability of their print-heads in terms of the number of characters printed before failure. To assess this, the manufacturer conducts a study on a small sample of print-heads due to the destructive nature of the testing process.

### A total of 15 print-heads were randomly selected and tested until failure. The durability of each print-head (in millions of characters) was recorded as follows:
1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29


In [ ]:
## a. Build 99% Confidence Interval Using Sample Standard Deviation

In [1]:
data=[1.13,1.55,1.43,0.92,1.25,1.36,1.32,0.85,1.07,1.48,1.20,1.33,1.18,1.22,1.29]

In [3]:
data

[1.13,
 1.55,
 1.43,
 0.92,
 1.25,
 1.36,
 1.32,
 0.85,
 1.07,
 1.48,
 1.2,
 1.33,
 1.18,
 1.22,
 1.29]

In [7]:
import scipy.stats as stats
import numpy as np

In [29]:
s=np.std(data)

In [31]:
s

0.18661427836285438

In [33]:
x_bar=np.mean(data)

In [35]:
x_bar

1.2386666666666666

In [45]:
## given data
n=15
## CI=0.99, alpha=0.01

In [47]:
t_critical=stats.t.ppf(0.995,14)

In [49]:
t_critical

2.976842734370834

In [75]:
margin_of_error1=t_critical*(s/np.sqrt(n))

In [77]:
margin_of_error1

0.14343499804054372

In [101]:
## lower limit
lower_limit=x_bar-margin_of_error1
lower_limit

1.095231668626123

In [103]:
## Upper limit
upper_limit=x_bar+margin_of_error1
upper_limit

1.3821016647072102

#### 99% confidence interval for the mean number of characters printed before the print-head fails from 1.09523 to 1.38210

In [107]:
## b. Build 99% Confidence Interval Using Known Population Standard Deviation

In [109]:
## Given data
pop_std=0.2

In [111]:
z_critical=stats.norm.ppf(0.995)

In [113]:
z_critical

2.5758293035489004

In [115]:
margin_of_error2=z_critical*(pop_std/np.sqrt(n))

In [117]:
margin_of_error2

0.13301525327090588

In [119]:
lower_limit=x_bar-margin_of_error2
lower_limit

1.1056514133957607

In [121]:
upper_limit=x_bar+margin_of_error2
upper_limit

1.3716819199375725

#### 99% confidence interval for the mean number of characters printed before failure if the population standard deviation is 0.2 million characters.from 1.10565 to 1.37168